src/demo-app/a11y/radio/radio-a11y.html

```diff
@@ -0,0 +1,32 @@
+<section>
+  <h2>Radio buttons in group</h2>
+  <md-radio-group name="seasons" [(ngModel)]="favoriteSeason">
```

Comment:
Each radio-group should have an `aria-label` (e.g. "Seasons") or an `aria-labelledby` ([example](https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/radio/radio.html))